src/lib.rs

```diff
@@ -691,6 +688,11 @@ impl<State> Key<State> {
         self.id
     }
 
+    /// Returns the `Commit` of the current `Revision`
+    pub fn commit_at_root(&self) -> Commit<State> {
```

Comment:
Let's call this method `current_value` -- `commit_at_root` was meant to describe the commit at the time when the state var was "bound" in the current revision.

Reply:
Seems a bit confusing if you would have `curent_value` while `latest` commits being passed to the "key updater".